import-scripts/clone_db_wrapper.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+# Script for running pre-import steps
+# Consists of the following:
+# - Determine which database is "production" vs "not production"
+# - Drop tables in the non-production database
+# - Clone the production database into the non-production database
+
+PORTAL_SCRIPTS_DIRECTORY=$1
+if [ -z $PORTAL_SCRIPTS_DIRECTORY ]; then
+    PORTAL_SCRIPTS_DIRECTORY="/data/portal-cron/scripts"
+fi
+if [ ! -f $PORTAL_SCRIPTS_DIRECTORY/automation-environment.sh ] ; then
+    echo "`date`: Unable to locate automation_env, exiting..."
+    exit 1
+fi
+source $PORTAL_SCRIPTS_DIRECTORY/automation-environment.sh
+
+# Create tmp directory for processing
+tmp=$PORTAL_HOME/tmp/import-cron-genie
+if ! [ -d "$tmp" ] ; then
+    if ! mkdir -p "$tmp" ; then
+        echo "Error : could not create tmp directory '$tmp'" >&2
+        exit 1
+    fi
+fi
+if [[ -d "$tmp" && "$tmp" != "/" ]]; then
+    rm -rf "$tmp"/*
+fi
+
+# This file will be used throughout entire import job to determine directionality
+SET_UPDATE_PROCESS_SCRIPT_FILEPATH="$PORTAL_SCRIPTS_DIRECTORY/set_update_process_state.sh"
+MANAGE_DATABASE_TOOL_PROPERTIES_FILEPATH="$PORTAL_SCRIPTS_DIRECTORY/airflowdb.properties"
```

Comment:
this properties file is genie specific, and also is specific to managing the dataflow tools needed for clickhouse processing. I think the filename should include additional specification. Also, I do not think the file containing the database credentials should reside in the scripts directory. The more conventional location for a properties file such as that would be in /data/portal-cron/pipeline-credentials (or perhaps /data/portal-cron/git-repos/portal-configuration)